Content.Client/Ghost/GhostToggleSelfVisibility.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Content.Shared.Ghost;
+using Robust.Client.GameObjects;
+using Robust.Shared.Console;
+
+namespace Content.Client.Ghost;
+
+public sealed class GhostToggleSelfVisibility : IConsoleCommand
```

Comment:
Make it a toolshedcommand

Reply:
Has to work without client. So has it be an IConsoleCommand. I talked to moony about it already.